### PR TITLE
chore: update to latest wallet and adapters

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- includes universal wallet migration updates from wallet and adapters

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

**Review by talwinder50 — APPROVED** [review]

**Review by fqutishat — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

File: adapter-issuer/kustomize/adapter-issuer/base/deployment.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ spec:
           - secretRef:
               name: issuer-env-secret
         image: issuer-adapter:latest
+        imagePullPolicy: IfNotPresent
```
Comment (rolsonquadras) [inline adapter-issuer/kustomize/adapter-issuer/base/deployment.yml:30]: Do we need this change ?

Comment (sudeshrshetty) [reply adapter-issuer/kustomize/adapter-issuer/base/deployment.yml:30]: missing in some projects and it doesn't update images when you have new one. Problem arises only when you use 'latest' image.

Comment (talwinder50) [reply adapter-issuer/kustomize/adapter-issuer/base/deployment.yml:30]: Make deploy inside wallet-web or inside k8s folder fails ... to pull the image. This seems to fix that issue

Comment (sudeshrshetty) [reply adapter-issuer/kustomize/adapter-issuer/base/deployment.yml]: removed it

**Pull request merged**